git-updater/tasks/main.yml

```diff
@@ -20,3 +20,11 @@
     job: "{{ SCRIPTS_DIR }}/git_updater.sh {{ item }} {{ LOG_DIR }}"
     user: "{{ USERNAME }}"
   loop: "{{ git_directories }}"
+- name: Ensure diskspace is cleaned up every 15 minutes, from 6am to 9pm
```

Comment:
I think we want to add a similar step as "Ensure git_update scripts are in place" to ensure that the cleanup script is in place. Or don't we need that?

Reply:
done in https://github.com/wmde/ansible-roles/pull/2/commits/f6589a795b7e46ea7cb8450ba3cc148de0dfeabc